etc/entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
Make sure this file is executable i.e. `chmod +x` it.

Reply:
-rwxr-xr-x 1 root root 776 Jul 30 21:30 entrypoint.sh